idsvr/templates/post-hook.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
     "helm.sh/hook-weight": "10"
+    {{- if .Values.curity.jobsAnnotations }}
```

Comment:
post-hook has its own settings options in the values file, can you instead use `postHook.annotations` instead?